tests/chainer_tests/distributions_tests/test_transformed_distribution.py

```diff
@@ -0,0 +1,41 @@
+from chainer import distributions
+from chainer import testing
+import numpy
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(3, 2), (1,)],
+    'is_variable': [True, False],
+    'sample_shape': [(3, 2), ()],
+}))
+@testing.fix_random()
+@testing.with_requires('scipy')
+class TestLogTransformedDistribution(testing.distribution_unittest):
```

Comment:
This class tests `Exp`-transformed `Normal` distribution, which is a log-normal distribution.